packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -75,7 +75,12 @@ export type SuiteAction =
           payload: Partial<AutodetectSettings>;
       }
     | { type: typeof deviceActions.requestDeviceReconnect.type }
-    | { type: typeof SUITE.SET_EXPERIMENTAL_FEATURES; payload?: ExperimentalFeature[] }
+    | {
+          type: typeof SUITE.SET_EXPERIMENTAL_FEATURES;
+          payload: {
+              enabledFeatures: ExperimentalFeature[] | undefined; // undefined means the experimental features are off as a whole
```

Comment:
Is there a reason for this change, or do you just want to be consistent with none of the inconsistent actions above? :upside_down_face:

Reply:
This allows future extension. We started doing this so future changes are not pain. For consistency reasons we are trying to use it everywhere in the same manner ( @jvaclavik  & @juriczech I think)

Reply:
I personally can't see an advantage of doing it in case I'll need it someday instead of doing it when I need it, but whatever. :+1:

Reply:
I see a difference here, but lets not prolong/complicate the discussion here. We can talk this more in person someday. :)